packages/react-dom/src/client/ReactDOMFiberComponent.js

```diff
@@ -378,6 +378,12 @@ export function createElement(
       // See discussion in https://github.com/facebook/react/pull/6896
       // and discussion in https://bugzilla.mozilla.org/show_bug.cgi?id=1276240
       domElement = ownerDocument.createElement(type);
+      // Make sure that `select` has `multiple` attribute set to appropriate state before appending options
+      // To prevent first option be initialy made selected
+      // see more details in https://github.com/facebook/react/issues/13222
+      if (type === 'select' && !!props.multiple) {
+        domElement.setAttribute('multiple', 'true');
```

Comment:
We use property assignment in [the postMountWrapper](https://github.com/facebook/react/blob/master/packages/react-dom/src/client/ReactDOMFiberSelect.js#L176), could you do the same here?

Reply:
`postMountWrapper` attempts to update options, but as at this moment select doesn't have any child appended, it makes no sense, it will iterate over empty options collection. However it should not cause any problems so if you think it is better approach I can do this change.